android/app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,15 @@
         android:name="android.hardware.camera.autofocus"
         android:required="false" />
 
+    <!-- Usage of @react-native-community/netinfo, rn-fetch-blob and Flipper
+         adds the permission requirement for ACCESS_WIFI_STATE in the final 
+         AndroidManifest of the app which makes wifi hardware a compulsary 
```

Comment:
Small comment/doc-writing tip: better to leave this less specific, just saying that some of our dependencies add this, rather than naming which ones in particular.

The reason is that the exact list may change over time: a future version of one of these may stop doing so, a future version of another dependency may start; we might drop one of these, or follow it across a rename, or pull in a new dependency that does the same thing too. When the comment is very specific like this, any of those changes will make it untrue and potentially confusing to a future reader -- *but*, for the point it's really trying to make, none of those details matter, only the fact that there's any dependency at all that does this. So that makes the point just as well, and is much more likely to remain true over time.

OTOH in the *commit message* these details are great to include (as you do) -- they explain your/our thinking at the time we're making this change, which can be helpful to a future reader; and as part of the commit message they're inherently contextualized as being about this particular moment in time. There's a bit more on this theme in our style guide:
https://github.com/zulip/zulip-mobile/blob/master/docs/style.md#commit-messages-vs-comments

Reply:
I see, will keep this in mind.